Error during build stage while deploying to Bluemix

![untitled](https://cloud.githubusercontent.com/assets/4971240/12360990/57651202-bbe1-11e5-8370-36ce72f11549.png)


**Here is the complete log**

Cloning the 'master' branch from repo 'https://P90MSG01:3041/git/ladwa/MeanJSladwa'
gpg: directory `/home/pipeline/.gnupg' created
gpg: new configuration file `/home/pipeline/.gnupg/gpg.conf' created
gpg: WARNING: options in `/home/pipeline/.gnupg/gpg.conf' are not yet active during this run
gpg: keyring `/home/pipeline/.gnupg/secring.gpg' created
gpg: keyring `/home/pipeline/.gnupg/pubring.gpg' created
gpg: requesting key D39DC0E3 from hkp server keys.gnupg.net
gpg: /home/pipeline/.gnupg/trustdb.gpg: trustdb created
gpg: key D39DC0E3: public key "Michal Papis (RVM signing) <mpapis@gmail.com>" imported
gpg: no ultimately trusted keys found
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
Downloading https://github.com/rvm/rvm/archive/1.26.11.tar.gz
Downloading https://github.com/rvm/rvm/releases/download/1.26.11/1.26.11.tar.gz.asc
gpg: Signature made Mon 30 Mar 2015 09:52:13 PM UTC using RSA key ID BF04FF17
gpg: Good signature from "Michal Papis (RVM signing) <mpapis@gmail.com>"
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 409B 6B17 96C2 7546 2A17  0311 3804 BB82 D39D C0E3
     Subkey fingerprint: 62C9 E5F4 DA30 0D94 AC36  166B E206 C29F BF04 FF17
GPG verified '/home/pipeline/.rvm/archives/rvm-1.26.11.tgz'

Installing RVM to /home/pipeline/.rvm/
    Adding rvm PATH line to /home/pipeline/.profile /home/pipeline/.mkshrc /home/pipeline/.bashrc /home/pipeline/.zshrc.
    Adding rvm loading line to /home/pipeline/.profile /home/pipeline/.bash_profile /home/pipeline/.zlogin.
Installation of RVM in /home/pipeline/.rvm/ is almost complete:

  * To start using RVM you need to run `source /home/pipeline/.rvm/scripts/rvm`
    in all your open shell windows, in rare cases you need to reopen all shell windows.

# pipeline,
#
#   Thank you for using RVM!
#   We sincerely hope that RVM helps to make your life easier and more enjoyable!!!
#
# ~Wayne, Michal & team.

In case of problems: http://rvm.io/help and https://twitter.com/rvm_io
rvm 1.26.11 (latest) by Wayne E. Seguin <wayneeseguin@gmail.com>, Michal Papis <mpapis@gmail.com> [https://rvm.io/]
Searching for binary rubies, this might take some time.
Found remote file https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.04/x86_64/ruby-2.2.1.tar.bz2
Checking requirements for ubuntu.
Installing requirements for ubuntu.
Updating system.....
Installing required packages: gawk, libreadline6-dev, zlib1g-dev, libssl-dev, libyaml-dev, libsqlite3-dev, sqlite3, autoconf, libgdbm-dev, libncurses5-dev, automake, libtool, bison, pkg-config, libffi-dev...................
Requirements installation successful.
ruby-2.2.1 - #configure
ruby-2.2.1 - #download
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 23.2M    0 81920    0     0   194k      0  0:02:02 --:--:--  0:02:02  194k
  1 23.2M    1  448k    0     0   323k      0  0:01:13  0:00:01  0:01:12  323k
  3 23.2M    3  880k    0     0   370k      0  0:01:04  0:00:02  0:01:02  370k
  6 23.2M    6 1568k    0     0   463k      0  0:00:51  0:00:03  0:00:48  463k
  9 23.2M    9 2336k    0     0   533k      0  0:00:44  0:00:04  0:00:40  533k
 13 23.2M   13 3264k    0     0   607k      0  0:00:39  0:00:05  0:00:34  641k
 18 23.2M   18 4304k    0     0   674k      0  0:00:35  0:00:06  0:00:29  771k
 27 23.2M   27 6672k    0     0   904k      0  0:00:26  0:00:07  0:00:19 1157k
 41 23.2M   41  9.7M    0     0  1191k      0  0:00:20  0:00:08  0:00:12 1681k
 53 23.2M   53 12.4M    0     0  1362k      0  0:00:17  0:00:09  0:00:08 2088k
 64 23.2M   64 15.1M    0     0  1494k      0  0:00:15  0:00:10  0:00:05 2448k
 66 23.2M   66 15.4M    0     0  1309k      0  0:00:18  0:00:12  0:00:06 2020k
 80 23.2M   80 18.6M    0     0  1543k      0  0:00:15  0:00:12  0:00:03 2486k
 89 23.2M   89 20.7M    0     0  1587k      0  0:00:15  0:00:13  0:00:02 2254k
 99 23.2M   99 23.2M    0     0  1654k      0  0:00:14  0:00:14 --:--:-- 2202k
100 23.2M  100 23.2M    0     0  1656k      0  0:00:14  0:00:14 --:--:-- 2076k
ruby-2.2.1 - #validate archive
ruby-2.2.1 - #extract
ruby-2.2.1 - #validate binary
ruby-2.2.1 - #setup
ruby-2.2.1 - #gemset created /home/pipeline/.rvm/gems/ruby-2.2.1@global
ruby-2.2.1 - #importing gemset /home/pipeline/.rvm/gemsets/global.gems.............................
ruby-2.2.1 - #generating global wrappers........
ruby-2.2.1 - #gemset created /home/pipeline/.rvm/gems/ruby-2.2.1
ruby-2.2.1 - #importing gemsetfile /home/pipeline/.rvm/gemsets/default.gems evaluated to empty gem list
ruby-2.2.1 - #generating default wrappers........
Creating alias default for ruby-2.2.1...
Successfully installed sass-3.4.21
Parsing documentation for sass-3.4.21
Installing ri documentation for sass-3.4.21
Done installing documentation for sass after 11 seconds
1 gem installed

  * To start using RVM you need to run `source /home/pipeline/.rvm/scripts/rvm`
    in all your open shell windows, in rare cases you need to reopen all shell windows.
npm WARN engine imagemin@4.0.0: wanted: {"node":">=0.10.0","npm":">=2.1.5"} (current: {"node":"0.10.40","npm":"2.1.1"})
npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated lodash@0.9.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
npm WARN optional dep failed, continuing fsevents@1.0.6
npm WARN optional dep failed, continuing fsevents@1.0.6
npm WARN engine escodegen@1.7.1: wanted: {"node":">=0.12.0"} (current: {"node":"0.10.40","npm":"2.1.1"})
npm WARN engine escodegen@1.7.1: wanted: {"node":">=0.12.0"} (current: {"node":"0.10.40","npm":"2.1.1"})
npm WARN optional dep failed, continuing fsevents@1.0.6
npm WARN engine csso@1.4.4: wanted: {"node":">=0.12.0"} (current: {"node":"0.10.40","npm":"2.1.1"})

> pngquant-bin@3.0.1 postinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/imagemin-pngquant/node_modules/pngquant-bin
> node lib/install.js

  ✔ pngquant pre-build test passed successfully

> optipng-bin@3.0.4 postinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-optipng/node_modules/optipng-bin
> node lib/install.js

  ✔ optipng pre-build test passed successfully

> gifsicle@3.0.3 postinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-gifsicle/node_modules/gifsicle
> node lib/install.js

  ✔ gifsicle pre-build test passed successfully

> jpegtran-bin@3.0.6 postinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-imagemin/node_modules/imagemin/node_modules/imagemin-jpegtran/node_modules/jpegtran-bin
> node lib/install.js

  ✔ jpegtran pre-build test passed successfully

> phantomjs@1.9.19 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/phantomjs
> node install.js

Looks like an `npm install -g`; unable to check for already installed version.
Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-x86_64.tar.bz2
Saving to /tmp/phantomjs/phantomjs-1.9.8-linux-x86_64.tar.bz2
Receiving...

Received 12854K total.
Extracting tar contents (via spawned process)
Removing /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/phantomjs/lib/phantom
Copying extracted folder /tmp/phantomjs/phantomjs-1.9.8-linux-x86_64.tar.bz2-extract-1452880910586/phantomjs-1.9.8-linux-x86_64 -> /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/phantomjs/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/phantomjs/lib/phantom/bin/phantomjs
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN engine selenium-webdriver@2.47.0: wanted: {"node":">= 0.12.x"} (current: {"node":"0.10.40","npm":"2.1.1"})
npm WARN engine selenium-webdriver@2.47.0: wanted: {"node":">= 0.12.x"} (current: {"node":"0.10.40","npm":"2.1.1"})

> spawn-sync@1.0.15 postinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-sass/node_modules/node-sass/node_modules/cross-spawn/node_modules/spawn-sync
> node postinstall

Installing native dependencies (this may take up to a minute)

> kerberos@0.0.17 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/connect-mongo/node_modules/mongodb/node_modules/kerberos
> (node-gyp rebuild) || (exit 0)

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/connect-mongo/node_modules/mongodb/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
In file included from ../lib/kerberos.cc:1:0:
../lib/kerberos.h:5:27: fatal error: gssapi/gssapi.h: No such file or directory
 #include <gssapi/gssapi.h>
                           ^
compilation terminated.
make: *** [Release/obj.target/kerberos/lib/kerberos.o] Error 1
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/connect-mongo/node_modules/mongodb/node_modules/kerberos/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/pipeline/.npm-global/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Linux 3.13.0-52-generic
gyp ERR! command "node" "/home/pipeline/.npm-global/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/connect-mongo/node_modules/mongodb/node_modules/kerberos
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 

> kerberos@0.0.17 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/acl/node_modules/mongodb/node_modules/kerberos
> (node-gyp rebuild) || (exit 0)

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/acl/node_modules/mongodb/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
In file included from ../lib/kerberos.cc:1:0:
../lib/kerberos.h:5:27: fatal error: gssapi/gssapi.h: No such file or directory
 #include <gssapi/gssapi.h>
                           ^
compilation terminated.
make: *** [Release/obj.target/kerberos/lib/kerberos.o] Error 1
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/acl/node_modules/mongodb/node_modules/kerberos/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/pipeline/.npm-global/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Linux 3.13.0-52-generic
gyp ERR! command "node" "/home/pipeline/.npm-global/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/acl/node_modules/mongodb/node_modules/kerberos
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 

> kerberos@0.0.17 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos
> (node-gyp rebuild) || (exit 0)

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
In file included from ../lib/kerberos.cc:1:0:
../lib/kerberos.h:5:27: fatal error: gssapi/gssapi.h: No such file or directory
 #include <gssapi/gssapi.h>
                           ^
compilation terminated.
make: *** [Release/obj.target/kerberos/lib/kerberos.o] Error 1
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/pipeline/.npm-global/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Linux 3.13.0-52-generic
gyp ERR! command "node" "/home/pipeline/.npm-global/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 
npm WARN deprecated static-favicon@1.0.2: use serve-favicon module

> node-sass@3.4.2 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-sass/node_modules/node-sass
> node scripts/install.js

Binary downloaded and installed at /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-sass/node_modules/node-sass/vendor/linux-x64-11/binding.node

> node-sass@3.4.2 postinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-sass/node_modules/node-sass
> node scripts/build.js

` /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-sass/node_modules/node-sass/vendor/linux-x64-11/binding.node ` exists. 
 testing binary.
Binary is fine; exiting.

> bufferutil@1.2.1 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/ws/node_modules/bufferutil/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node: Finished
  COPY Release/bufferutil.node
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/ws/node_modules/bufferutil/build'

> utf-8-validate@1.2.1 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/ws/node_modules/utf-8-validate/build'
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/obj.target/validation.node
  SOLINK_MODULE(target) Release/obj.target/validation.node: Finished
  COPY Release/validation.node
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/ws/node_modules/utf-8-validate/build'

> v8-profiler@5.5.0 preinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-profiler
> node -e 'process.exit(0)'


> v8-debug@0.7.0 preinstall /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug
> node -e 'process.exit(0)'


> bufferutil@1.2.1 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/bufferutil/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node: Finished
  COPY Release/bufferutil.node
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/bufferutil/build'

> utf-8-validate@1.2.1 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate/build'
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/obj.target/validation.node
  SOLINK_MODULE(target) Release/obj.target/validation.node: Finished
  COPY Release/validation.node
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate/build'

> bufferutil@1.2.1 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-protractor/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-protractor/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/bufferutil/build'
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node: Finished
  COPY Release/bufferutil.node
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-protractor/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/bufferutil/build'

> utf-8-validate@1.2.1 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-protractor/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

make: Entering directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-protractor/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate/build'
  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/obj.target/validation.node
  SOLINK_MODULE(target) Release/obj.target/validation.node: Finished
  COPY Release/validation.node
make: Leaving directory `/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/gulp-protractor/node_modules/protractor/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate/build'

> v8-debug@0.7.0 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug
> node-pre-gyp install --fallback-to-build

[v8-debug] Success: "/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-debug/build/debug/v0.7.0/node-v11-linux-x64/debug.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile

> v8-profiler@5.5.0 install /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-profiler
> node-pre-gyp install --fallback-to-build

[v8-profiler] Success: "/home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/node_modules/grunt-node-inspector/node_modules/node-inspector/node_modules/v8-profiler/build/profiler/v5.5.0/node-v11-linux-x64/profiler.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile
npm ERR! Linux 3.13.0-52-generic
npm ERR! argv "node" "/home/pipeline/.npm-global/bin/npm" "install"
npm ERR! node v0.10.40
npm ERR! npm  v2.1.1
npm ERR! path /tmp/npm-3496-pW2MRkkY/registry.npmjs.org/mime-types/-/mime-types-2.0.14.tgz-unpack
npm ERR! code ENOTEMPTY
npm ERR! errno 53

npm ERR! ENOTEMPTY, rmdir '/tmp/npm-3496-pW2MRkkY/registry.npmjs.org/mime-types/-/mime-types-2.0.14.tgz-unpack'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/pipeline/b693b87a-eb3a-4a37-88c5-611b99daae41/npm-debug.log

Finished: ERRORED

--- a/.bluemix/pipeline.yml
+++ b/.bluemix/pipeline.yml
@@ -13,6 +13,8 @@ stages:
     build_type: grunt
     script: |-
       #!/bin/bash
+      # Set Node version to 0.12
+      export PATH=/opt/IBM/node-v0.12/bin:$PATH
       # Install RVM, Ruby, and SASS
       # Needed when running grunt build
       gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3

--- a/package.json
+++ b/package.json
@@ -47,7 +47,7 @@
     "method-override": "~2.3.5",
     "gulp-wiredep": "~0.0.0",
     "mocha": "~2.4.5",
-    "mongoose": "~4.4.3",
+    "mongoose": "~4.4.3 <4.4.7",
     "morgan": "~1.6.1",
     "multer": "~1.1.0",
     "nodemailer": "~2.1.0",

